.github/workflows/linting.yml

```diff
@@ -0,0 +1,24 @@
+name: Code quality check
+
+on: [push]
+
+jobs:
+  lint:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install apt dependencies
+      run: |
+        sudo apt-get update && sudo apt-get install python3 python3-pip -y
+    - name: Install pip dependencies
+      run: |
+        pip3 install flake8 pylint
+    - name: Lint with Flake8
+      run: |
+        # E402: module level import not at top of file
+        flake8 --ignore=E265,F821,E501,W503,E203,E402 --max-line-length=88 --count --statistics --show-source .
```

Comment:
Here flake has `--max-line-length=88` while in `.github/workflows/black.yml` it is `--line-length 79` - should that be sync'ed?